src/pages/ProjectSettings.tsx

```diff
@@ -0,0 +1,127 @@
+import styled from "@emotion/styled";
+import { SideNav, SideNavGroup, SideNavItem } from "@leafygreen-ui/side-nav";
+import { useParams, Link, Redirect } from "react-router-dom";
+import { PageWrapper } from "components/styles";
+import {
+  ProjectSettingsTabRoutes,
+  getProjectSettingsRoute,
+} from "constants/routes";
+import { usePageTitle } from "hooks";
+import { environmentalVariables } from "utils";
+import {
+  ProjectSettingsTabs,
+  getTitle,
+} from "./projectSettings/ProjectSettingsTabs";
+
+const { isProduction } = environmentalVariables;
```

Comment:
This checks if it's "production", but at what point does this page get set to "development"?

Reply:
The [`isProduction()`](https://github.com/evergreen-ci/spruce/blob/main/src/utils/environmentalVariables.ts#L21-L22) util uses the `process.env.NODE_ENV` environment variable, which is set [here](https://github.com/evergreen-ci/spruce/blob/main/scripts/build.js#L5) for prod and [here](https://github.com/evergreen-ci/spruce/blob/main/scripts/start.js#L5) for dev.

Reply:
Ah, I was misreading what this is doing. It makes sense, thanks!

Reply:
This happens in the build step for deploys this variable gets set to true automatically.
When developing locally this variable will be false.